.github/workflows/ci.yml

```diff
@@ -79,7 +79,7 @@ jobs:
 
             - name: Install Pallene Tracer
               run: |
-                  git clone https://github.com/pallene-lang/pallene-tracer
+                  git clone --depth 1 https://github.com/pallene-lang/pallene-tracer --branch 0.5.0a
```

Comment:
Consider using a variable for the Tracer version, similarly to what we already do for the Lua version.